### PR TITLE
Defibox_vault.md

### PR DESCRIPTION
**Pull request by DefiboxTeam**
### Project Abstract

> Please provide a brief description of your project here summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.
> Please delete these lines that start with a `>`

### For which grant [level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels) are you applying?

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $50,000, 3 approvals
- [x] **Level 3**:  Unlimited, 4 approvals

### Application Checklist

- [x] The [application template]([xxxx](https://github.com/eosnetworkfoundation/grant-framework/blob/main/applications/application-template.md)) has been copied, renamed (`my_project_name.md`) and updated.
- [x] The application file (`my_project_name.md`) has no spaces in its name, ends with the `.md` extension has been placed in the `applications` directory.
- [x] I have read and understood the:
  - [ ] [FAQs](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/faq.md), 
  - [ ] [application guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/grant_guidelines_per_category.md) and 
  - [ ] [announcement guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/announcement-guidelines.md).
- [x] An EOS address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledge the [terms and conditions](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] This project has NOT been part of a token sale or NFT sale.
- [ ] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/eosnetworkfoundation/grant-milestones#process).

### How Did You Hear About our grants program?

- [ ] Social Media
- [x] EOSNetwork.com
- [x] Personal Recommendation
- [ ] ENF Fireside Chat
- [ ] Investor/VC
- [ ] Online Search
- [ ] Hackathon
- [ ] Other: _______

### CODE REVIEW

**Comment by tedcahalleos** [comment]
As stated repeatedly in email, this project must be fully open-source from the beginning.  The grant evaluators will need to look at the source code for each milestone and sign off before payment.  Failure to make this project fully open-source from the start will result in the application being rejected.

**Comment by DefiboxTeam** [comment]
> As stated repeatedly in email, this project must be fully open-source from the beginning. The grant evaluators will need to look at the source code for each milestone and sign off before payment. Failure to make this project fully open-source from the start will result in the application being rejected.

Thanks for the feedback, we will see how we can proceed this

**Comment by tedcahalleos** [comment]
The grant committee has approved the grant.  You still need to work with the Grant Evaluators to meet their requirements for the grant to be fully approved and the work on Milestones to begin.

**Comment by DefiboxTeam** [comment]
Thanks for the approval, I will keep on the next steps




&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: &nbsp;"Ted&nbsp;Cahall&nbsp;@***@***.***&gt;;
Date: &nbsp;Fri, Jan 20, 2023 02:31 AM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [eosnetworkfoundation/grant-framework] Defibox_vault.md (PR #98)

&nbsp;



 
The grant committee has approved the grant.  You still need to work with the Grant Evaluators to meet their requirements for the grant to be fully approved and the work on Milestones to begin.

**Review by DenisCarriere — APPROVED** [review]
## To change
- [ ] Update project repo link

## Deliverables
The proposed deliverables are technically achievable by the Defibox's development team.

Project provides sufficient documentation to review the overall scope of the smart contract.

FYI: For other reviewers, smart contract is currently deployed https://bloks.io/account/vault.defi

File: applications/Defibox_vault.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
+# EOS Network Foundation Grant Proposal
+
+- **Project Name:** Vault Protocol
+- **Team Name:** Defibox
+- **EOS Payment Address:** liquid.defi
+- **[Level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels):** 3
+- **Pomelo Grant(s):** https://pomelo.io/grants/vault
+- **Project is Open-Source:** Yes*
+- **Project was part of Token sale:** No
+- **Repository where Project resides:** https://github.com/DefiboxTeam/DefiboxVault-interface
```
Comment (DenisCarriere) [inline applications/Defibox_vault.md:10]: The project repo referenced here is empty

I'm assuming it should be the following URL: https://github.com/defiboxswap/DefiboxVault-interface

Comment (larryk85) [reply applications/Defibox_vault.md:10]: Currently, there is no license in this repo. Which license are you planning on using?

**Review by larryk85 — CHANGES_REQUESTED** [review]

File: applications/Defibox_vault.md (hunk 1 above)
Comment (DefiboxTeam) [reply applications/Defibox_vault.md:10]: Appropriate license has been added.
https://github.com/defiboxswap/DefiboxVault-interface/blob/main/LICENSE

**Review by larryk85 — APPROVED** [review]

**Review by DefiboxTeam — COMMENTED** [review]
Updated project repo link.

**Comment by DefiboxTeam** [comment]
@spoonincode @wanderingbort 
Dear Reviewers:

Greetings. I'm Raven from Defibox. I got to know that there are still reviewers that remaining to over check the grant. Thank you for your support for Defibox. Our team is asking the progress of this Grant. We already updated the info needed, so if you have any question or feedback, please don't hesitate to contact me here. Highly appreciated if you can help us with this.

Thank you very much

Defibox Team

File: applications/Defibox_vault.md (hunk 1 above)
Comment (Mattijah) [reply applications/Defibox_vault.md]: Question: The repo only contains header files. There is no related code. How is the license even useful when only `hpp` will be made public?

Comment (DenisCarriere) [reply applications/Defibox_vault.md]: @Mattijah If you review the **Open source - Timelines** it indicates the Stage 2/3 will include the full code to be released open source.

For security reasons, the *.cpp is not made available in the first stage, as the vault contains an excess of 9M EOS.

File: applications/Defibox_vault.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+**Stage Two (not completed)**
+
+- [ ] Protocol Design Definition (translated in English/Korean/Chinese)
+- [ ] SC development (production)
+- [ ] User documentation
+- [ ] Automatic Contracts Tests
+
+**Stage Three (re-evaluation 2nd Security Audit from Certik) “Version v1.0.0”**
+
+- [ ] All smart contract source code will be made available (*.cpp)
```
Comment (DenisCarriere) [inline applications/Defibox_vault.md:42]: @Mattijah to answer your question here: https://github.com/eosnetworkfoundation/grant-framework/pull/98/files/e69fe19c14ec730df8c1bdb4a62869d4b29681d2#r1107862746

**Review by spoonincode — APPROVED** [review]
Apologizes for my delay in reviewing; everything looks good.

File: applications/Defibox_vault.md (hunk 2 above)
Comment (Mattijah) [reply applications/Defibox_vault.md:42]: > For security reasons, the *.cpp is not made available in the first stage, as the vault contains an excess of 9M EOS.

The vault will still contain the same amount of EOS even after the 1st stage, so I'm not sure how valid this argument is nevertheless if this is something that will be done then alright. Thanks for clarifying.

**Review by Mattijah — APPROVED** [review]

**Review by wanderingbort — APPROVED** [review]

**Comment by tedcahalleos** [comment]
Grant fully approved.  Please submit Milestones here: https://github.com/eosnetworkfoundation/grant-milestones

**Comment by DefiboxTeam** [comment]
@tedcahalleos Thanks for all the help. I already submit the invoice form and milestone files according to your e-mail link. Thanks

**Pull request merged**